doc/UsersGuide/BackgroundInfo/TechnicalOverview.rst

```diff
@@ -183,28 +183,42 @@ The ``ufs-srweather-app`` :term:`umbrella repository` is an NCO-compliant reposi
    │     ├── bash_utils
    │     ├── machine
    │     ├── wrappers
+   │     ├── python_utils
    │     ├── config.community.yaml
+   │     ├── config.*.yaml
    │     ├── config_defaults.yaml
-   │     ├── experiment.jsonschema
    │     ├── generate_FV3LAM_wflow.py
    │     ├── launch_FV3LAM_wflow.sh
-   │     └── setup.py
-   └── versions
-
-SRW App SubDirectories
+   │     ├── setup.py
+   │     └── valid_param_vals.yaml
```

Comment:
```suggestion
```
This file was removed in Christina's `uwtools` PR. There may be other files that were added or deleted, so it's best to check the current state of develop and make sure this diagram matches that.